test/jdk/javax/swing/JList/4618767/JListSelectedElementTest.java

```diff
@@ -0,0 +1,217 @@
+                    robot.keyRelease(KeyEvent.VK_CONTROL);
+                    robot.keyRelease(MENU);
+                }else{
+                    robot.keyPress(KeyEvent.VK_ALT);
+                    robot.keyPress(MENU);
+                    robot.keyRelease(KeyEvent.VK_ALT);
+                    robot.keyRelease(MENU);
+                }
+
+                waitCount = 0;
+                while (!menuSelected) {
+                    robot.delay(100);
+                    waitCount++;
+                    if (waitCount > 20) {
+                        throw new RuntimeException("Can't select menu using mnemonics for " + laf);
+                    }
+                }
+
+                robot.keyPress(KeyEvent.VK_ENTER);
+                robot.keyRelease(KeyEvent.VK_ENTER);
+
+                String elementSelAfter = list.getSelectedValue();
+
+                // As per the fix of BugID 4618767, the list element selection should not change
+                if (!elementSel.equals(elementSelAfter)) {
+                    throw new RuntimeException("Test failed for " + laf
+                            + " as list.getSelectedValue() before = " + elementSel
+                            + " not equal to list.getSelectedValue() after pressing Enter = " + elementSelAfter
+                    );
+                }
+                System.out.println("Test passed for laf: " + laf);
+
+            } finally {
+                SwingUtilities.invokeAndWait(JListSelectedElementTest::disposeFrame);
+            }
+        }
+    }
+
+    private static boolean isFocusOwner() throws Exception {
+        AtomicBoolean isFocusOwner = new AtomicBoolean(false);
+        SwingUtilities.invokeAndWait(() -> isFocusOwner.set(list.isFocusOwner()));
+        return isFocusOwner.get();
+    }
+
+    private static void createUI() {
+        frame = new JFrame();
+        list = new JList(new String[]{"anaheim", "bill", "chicago", "dingo", "ernie", "freak"});
+        JMenu menu = new JMenu("File");
+        menu.setMnemonic(MENU);
+        JMenuItem menuItem = new JMenuItem("Dummy");
+        menu.add(menuItem);
+        menu.addMenuListener(new MenuListener() {
+                                 @Override
+                                 public void menuSelected(MenuEvent e) {
+                                     menuSelected = true;
+                                 }
+                                 @Override
+                                 public void menuDeselected(MenuEvent e) {
+                                 }
+                                 @Override
+                                 public void menuCanceled(MenuEvent e) {
+                                 }
+                             }
+        );
+
+        JMenuBar menuBar = new JMenuBar();
+        menuBar.add(menu);
+
+        frame.setJMenuBar(menuBar);
+        frame.setContentPane(list);
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
```

Comment:
```suggestion
        frame.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
```
You should never call `System.exit` from jtreg tests.